core/src/main/java/org/eclipse/hono/util/TenantObject.java

```diff
@@ -240,24 +232,27 @@ public X509Certificate getTrustedCertificateAuthority() throws CertificateExcept
      * @return The trust anchor or {@code null} if no trusted certificate authority
      *         has been set.
      * @throws GeneralSecurityException if the value of the <em>trusted-ca</em> property
-     *             cannot be parsed into a trust anchor, e.g. because of unsupported
-     *             key type, malformed certificate or public key encoding etc. 
+     *              cannot be parsed into a trust anchor, e.g. because of unsupported
+     *              key type, malformed certificate or public key encoding etc.
      */
     @JsonIgnore
-    public TrustAnchor getTrustAnchor() throws GeneralSecurityException {
-
-        if (trustAnchor != null) {
-            return trustAnchor;
+    public List<TrustAnchor> getTrustAnchors() throws GeneralSecurityException {
+        if (trustAnchors != null) {
+            return trustAnchors;
```

Comment:
this means that if I add another trust anchor after having invoked this method, the newly added trust anchor will not be contained in the returned result, right?

Reply:
yes.

> if I add another trust anchor after having invoked this method

but you usually will not do this right? this method is called when the tenant data stored in the registry has been retrieved via the TenantService

Reply:
that is one way of using this class, yes. However, since it is part of Hono's public API, we should not make any assumptions regarding how it will be used. Can you please change it so that the result returned by this method is will consistently return the current trust anchors?

Reply:
> Can you please change it so that the result returned by this method is will consistently return the current trust anchors?

yes, I will